readthedocs/projects/models.py

```diff
@@ -562,23 +561,6 @@ def get_builds_url(self):
             },
         )
 
-    def get_canonical_url(self):
```

Comment:
Curious how you found this, were you looking at everywhere DONT_HIT_ settings were used?

Reply:
Checking everywhere where we were using the API, and adapting the ones that were used, and removing the ones that weren't used in our code.